device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/platform_reboot

```diff
@@ -1,31 +1,30 @@
 #!/bin/bash
 
-update_reboot_cause_for_supervisor_reboot()
+update_reboot_cause()
 {
     DEVICE_MGR_REBOOT_FILE=/tmp/device_mgr_reboot
     REBOOT_CAUSE_FILE=/host/reboot-cause/reboot-cause.txt
-    TMP_REBOOT_CAUSE_FILE=/tmp/tmp-reboot-cause.txt
-    if [ -f  $DEVICE_MGR_REBOOT_FILE ]; then
-        if [ -f $REBOOT_CAUSE_FILE ]; then
-            t1=`sudo grep "User: ," $REBOOT_CAUSE_FILE`
-            if [ ! -z "$t1" ]; then
-                echo $t1 | sed 's/reboot/reboot from Supervisor/g' | sed 's/User: /User: admin/g' > $TMP_REBOOT_CAUSE_FILE
-                cp $TMP_REBOOT_CAUSE_FILE $REBOOT_CAUSE_FILE
-            fi
+    DEVICE_REBOOT_CAUSE_FILE=/etc/opt/srlinux/reboot-cause.txt
+    if [ -e  $DEVICE_MGR_REBOOT_FILE ]; then
+        if [ -e $DEVICE_REBOOT_CAUSE_FILE ]; then
+            cp -f $DEVICE_REBOOT_CAUSE_FILE $REBOOT_CAUSE_FILE
```

Comment:
Does this change need a new NDK for this change ?

Reply:
I see the ndk already merged in https://github.com/Azure/sonic-buildimage-msft/pull/77, will go ahead and merge + mark for 202205